### ISSUE TITLE
connect minio artifact store to persistent storage

### ISSUE THREAD

**Issue by trey-stafford** [QGreenland-Net/ogdc-helm#11]

**Comment by mbjones** [comment]
Again, be careful with how much multiple stages of the workflow depend on a single shared storage system. We found that I/O wait on shared storage was a very real, debilitating bottleneck in the ice wedge polygon runs for the viz-workflow. Less of an issue for smaller datasets though.

**Comment by trey-stafford** [comment]
> Again, be careful with how much multiple stages of the workflow depend on a single shared storage system. We found that I/O wait on shared storage was a very real, debilitating bottleneck in the ice wedge polygon runs for the viz-workflow. Less of an issue for smaller datasets though.

Yes, I'm thinking that particularly large datasets will need custom configurations to access storage for their particular workflows. We're working to support the majority of small/medium sized cases where we just need to e.g., get a dataset from an external source, reproject and subset,  and then publish the output. I suspect this is the interface most users would interact with.

I expect the really large dataset processing to be curated more directly by the QGreenland-Net/DataOne team, who will have more intimate knowledge over internal infrastructure architecture.

That said, we should certainly think about how we can abstract different storage options for various use-cases to make this as straightforward as possible.

**Comment by rmarow** [comment]
Rushiraj faced same issue as Robyn M - will be moving on from this due to final output persistent storage working
